.github/workflows/target-test.yml

```diff
@@ -197,6 +197,50 @@ jobs:
             ${{ env.TEST_DIR }}/build/config/sdkconfig.json
           if-no-files-found: error
 
+  build_esp_mqtt_cxx:
+    strategy:
+      matrix:
+        idf_ver: ["latest", "release-v5.0"]
+        idf_target: ["esp32"]
+    runs-on: ubuntu-20.04
+    container: espressif/idf:${{ matrix.idf_ver }}
+    env:
+      TEST_DIR: components/esp_mqtt_cxx/examples
+    steps:
+      - name: Checkout esp-protocols
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Build ${{ matrix.example }} with IDF-${{ matrix.idf_ver }} for ${{ matrix.idf_target }}
+        env:
+          IDF_TARGET: ${{ matrix.idf_target }}
+        shell: bash
+        working-directory: ${{ env.TEST_DIR }}
+        run: |
+          . ${IDF_PATH}/export.sh
+          cat sdkconfig.ci >> sdkconfig.defaults
```

Comment:
This is causing the CI to fail.
I'd suggest using `idf_build_apps.py` which can handle various configs and the default one, as well.

The other issue is in websocket client job @gabsuren 
This change should fix it:
```patch
diff --git a/components/esp_websocket_client/examples/CMakeLists.txt b/components/esp_websocket_client/examples/CMakeLists.txt
index 4f5b8aa..6394830 100644
--- a/components/esp_websocket_client/examples/CMakeLists.txt
+++ b/components/esp_websocket_client/examples/CMakeLists.txt
@@ -2,7 +2,7 @@
 # in this exact order for cmake to work correctly
 cmake_minimum_required(VERSION 3.5)
 
-set(EXTRA_COMPONENT_DIRS "../..")
+set(EXTRA_COMPONENT_DIRS "..")
 # This example uses an extra component for common functions such as Wi-Fi and Ethernet connection.
 list(APPEND EXTRA_COMPONENT_DIRS "../../../common_components/protocol_examples_common")
```